_posts/tsukori/2016-04-29-finalprojectupdate4.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+author: tsukori
+title: "Fourth Final Project Update" 
+--- 
+
+<iframe src="https://trinket.io/embed/python/cfd0381bba" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+So I currently have a lot of working pieces, they just need to get hooked together so they function correctly. The fun/frustrating part about this project is that a lot of stuff I thought I understood about python and python variables, I didn't so I'm having to think a lot more creatively and actually do what is "right" instead of trying to brute force things into working. My plan for getting all of the pieces and parts working together is to draw it out and write what needs to go where and what needs to be happening, plus I'm going to make a new trinket and reorganize my modules cause they are currently a giant mess. 
+
+I marked off a lot of milestones cause TECHNICALLY they are all working, I developed them outside of my current trinket and they all worked as intended, but together they don't, but I want to feel accomplished so I'm ticking them. Everything seems to be on schedule to finish right on time on Sunday so I'm looking forward to that. The brokenness roadblock that I am currently facing is frustrating but I think I will be able to make it through just fine. In the worst case scenario I will remove the ability of the user to add a word list of their own cause that's giving me the most problem actually. 
+
+I've never actually imported images into trinket so that will definitely be an interesting afternoon/evening of fun. What I'm working on next is finishing this project and making this hangman game a lot cooler than the original. So see you in 2 days. 
```

Comment:
Should be pretty straightforward.  Add them using the images tab then reference them as if they were in the sam folder as main.py, e.g. screen.register_shape("image.png") or what have you